x/tally/keeper/gas_meter.go

```diff
@@ -56,6 +57,7 @@ func (k Keeper) DistributionsFromGasMeter(ctx sdk.Context, reqID string, reqHeig
 	}
 
 	dists[0].Burn.Amount = dists[0].Burn.Amount.Add(reducedPayoutBurn.Mul(gasMeter.GasPrice()))
+	telemetry.SetGauge(float32(dists[0].Burn.Amount.Uint64()), "seda_tally_end_block_gas_burned")
```

Comment:
Just to check, does this add to the gauge for every invocation or set it to the value passed?

Reply:
Ah yea the prometheus version was doing an Add not a Set let me see if telemetry has an equivalent

Reply:
Hmmm we could convert it to seda then set it?

Reply:
Ok changes made as we discussed and tested it the way we discussed!